ouroborus/lib/ouroborus/container.rb

```diff
@@ -42,6 +43,19 @@ def daemon=(value)
       end
     end
 
+
+    def add_host?
+      !@add_host.empty?
+    end
+
+    def add_host=(value)
+      @add_host = value.to_s
+    end 
+
+    def teste
+      puts "teste >> exec branch jp"
+    end
```

Comment:
Was this method intentionally lefted here ?

Reply:
it's garbage

Reply:
fixed